Compilation fails if bsodmon is disabled

Since #1540 , compilation fails upon disabling bsodmon plugin with `--disable-plugin-bsodmon`. This is due to the following variable not being used anymore under those circumstances : https://github.com/tklengyel/drakvuf/blob/a2dc0203d5393829449feea0a3d0d28639f591e7/src/main.cpp#L426 

I suggest this PR gets reverted, until this variable is stored in an `options` instance, as every other plugin does.

## Error stack strace

```
clang++ -std=gnu++17 -DHAVE_CONFIG_H -I. -I..   -I/usr/local/include -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/json-c  -I.. -I.  -std=gnu++17 -fno-strict-overflow -D_FORTIFY_SOURCE=2 -O2 -fstack-protector-all -Wstack-protector --param ssp-buffer-size=1 -fPIE -DHARDENING  -DDISABLE_OUTPUT_BUFFERING  -DDRAKVUF_DEBUG -Werror -Wall -Wextra -g -ggdb3 -Wformat -Wformat-security -ferror-limit=0 -Wno-unknown-warning-option -Wno-c99-designator -Wno-reorder-init-list  -std=gnu++17 -MT main.o -MD -MP -MF $depbase.Tpo -c -o main.o main.cpp &&\
mv -f $depbase.Tpo $depbase.Po
main.cpp:439:10: error: unused variable 'bsodmon_ignore_stop' [-Werror,-Wunused-variable]
    bool bsodmon_ignore_stop = false;
```

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -423,8 +423,11 @@ int main(int argc, char** argv)
     bool procdump_on_finish = true;
     bool libdrakvuf_get_userid = true;
     std::set<uint64_t> ignored_processes;
-    bool bsodmon_ignore_stop = false;
     bool enable_active_callback_check = false;
+
+#ifdef ENABLE_PLUGIN_BSODMON
+    bool bsodmon_ignore_stop = false;
+#endif
 
     eprint_current_time();
 

